elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideDynamicConfiguration.java

```diff
@@ -0,0 +1,148 @@
+            String hibernateGetDDLAuto = hibernateProperties.getDdlAuto();
+
+            //Set the relevant property in JPA corresponding to Hibernate Property Value
+            if (jpaPropMap.get(HIBERNATE_DDL_AUTO) == null && hibernateGetDDLAuto != null) {
+               jpaPropMap.put(HIBERNATE_DDL_AUTO, hibernateGetDDLAuto);
+             }
+
+            ElideDynamicEntityCompiler compiler = dynamicCompiler.getIfAvailable();
+
+            Collection<ClassLoader> classLoaders = new ArrayList<>();
+            classLoaders.add(compiler.getClassLoader());
+
+            //Add dynamic classes to Pui Map
+            puiPropertyMap.put(AvailableSettings.CLASSLOADERS, classLoaders);
+            //Add classpath entity model classes to Pui Map
+            puiPropertyMap.put(AvailableSettings.LOADED_CLASSES, bindClasses);
+
+            //pui properties from pui map
+            Properties puiProps = new Properties();
+            puiProps.putAll(puiPropertyMap);
+
+            //Create Elide dynamic Persistence Unit
+            ElideDynamicPersistenceUnit elideDynamicPersistenceUnit =
+                    new ElideDynamicPersistenceUnit("dynamic", compiler.classNames, puiProps,
+                    compiler.getClassLoader());
+            elideDynamicPersistenceUnit.setNonJtaDataSource(source);
+            elideDynamicPersistenceUnit.setJtaDataSource(source);
+
+            HibernateJpaVendorAdapter vendorAdapter = new HibernateJpaVendorAdapter();
+            vendorAdapter.setShowSql(jpaProperties.isShowSql());
+            vendorAdapter.setGenerateDdl(jpaProperties.isGenerateDdl());
+            if (jpaProperties.getDatabase() != null) {
+                vendorAdapter.setDatabase(jpaProperties.getDatabase());
+            }
+            if (jpaProperties.getDatabasePlatform() != null) {
+                vendorAdapter.setDatabasePlatform(jpaProperties.getDatabasePlatform());
+            }
+
+            LocalContainerEntityManagerFactoryBean bean = new LocalContainerEntityManagerFactoryBean();
+            bean.setJpaVendorAdapter(vendorAdapter);
+
+            //Add JPA Properties from Application.yaml
+            bean.setJpaPropertyMap(jpaPropMap);
+
+            //Add Classes
+            bean.setJpaPropertyMap(puiPropertyMap);
+
+            bean.setPersistenceUnitManager(new PersistenceUnitManager() {
+                @Override
+                public PersistenceUnitInfo obtainDefaultPersistenceUnitInfo() throws IllegalStateException {
+                    return elideDynamicPersistenceUnit;
+                }
+
+                @Override
+                public PersistenceUnitInfo obtainPersistenceUnitInfo(String persistenceUnitName)
+                        throws IllegalArgumentException, IllegalStateException {
+                    return elideDynamicPersistenceUnit;
+                }
+            });
+
+            return bean;
+        } catch (Exception e) {
```

Comment:
Two things - let's not catch Exception here (it is too broad).  Also, if there is an error, we'll want to bring down the service.  Let the exceptions propagate.